_posts/2017-02-17-holacracy-and-purpose.md

```diff
@@ -0,0 +1,65 @@
+
+## Same situation, different approach
+
+Now, let's do this same experiment for a self-managing organization running on Holacracy. I assume familiarity with Holacracy's basic concepts. Otherwise, consider watching [this animation video](https://www.youtube.com/watch?v=MUHfVoQUj54) or [this talk by Brian Robertson](https://www.youtube.com/watch?v=tJxfJGo-vkI) to get a quick overview.
+
+For the context of this article, we'll focus on the concept of _Lead Link_. In the Holacracy operating system, there is no such thing as a "managing director" or other "manager" or "leader" job titles. Instead, work is divided over circles and roles. Each circle has a purpose and a special role called Lead Link. One of the Lead Link's responsibilities is to ensure the circle is working towards its purpose.
+
+The top-level circle in an organization is called the _Anchor Circle_. The purpose of that circle is also the organization's overall purpose. The Lead Link of the Anchor Circle is accountable for discovering and expressing this purpose.
+
+Let's apply this to our software development company. Instead of being managing director, you now fill the Lead Link role for the company's Anchor Circle. You can still get the same reports: Sales is doing great, Engineering is struggling. Based on this information, you can also still make the same strategic decision to focus on increasing engineering capacity.
+
+So far so good. But what's different then?
+
+As managing director, you could manage everything from strategy down to projects and actions. As a Lead Link, your authority stops after the strategic level. In a self-managing organization, you can no longer _tell_ your Recruitment department (which would technically be called a Recruitment circle in Holacracy terms) to take up a project to hire two new engineers. You can _ask_, but ultimately the person responsible for recruitment is the one deciding if and when the project is taken up.
+
+## The concept of purpose
+
+At first, this might feel odd. Why, from a managing director point of view, would you voluntarily choose to give up control over which projects have priority and how they are being executed? To answer this question, we'll first take a closer look at the concept of purpose.
+
+Holacracy [defines purpose as follows](http://www.holacracy.org/constitution#art523):
+
+> (...) the deepest creative potential [an organization] can sustainably express in the world, given all of the constraints acting upon it and everything available to it.
+
+As said earlier, as a company's top-most Lead Link, you are accountable for discovering and expressing the organization's purpose. This means you have a duty to ask this question: what is the deepest creative potential our organization can express in the world? Or put differently: what is the reason our organization exists?
```

Comment:
😄 Goed punt! Het leek me niet goed om daar op in te gaan, juist omdat het maar een gedachtenexperiment is, en het punt van het artikel niet is _wat_ precies het purpose is van dit voorbeeldbedrijf. Maar wat je zegt is ook waar: zoals het nu is, voelt het ook alsof er iets ontbreekt.